web3.js/src/transaction.ts

```diff
@@ -680,9 +713,13 @@ export class Transaction {
   static populate(
     message: Message,
     signatures: Array<string> = [],
+    lastValidBlockHeight?: LastValidBlockHeight,
   ): Transaction {
     const transaction = new Transaction();
     transaction.recentBlockhash = message.recentBlockhash;
+    if (lastValidBlockHeight) {
+      transaction.lastValidBlockHeight = lastValidBlockHeight;
+    }
```

Comment:
Why do you need to do this again? Isn't this value now part of the `Message` type?

Reply:
Hmm, yeah. I'll update this. Thanks!